.vscode/settings.json

```diff
@@ -60,5 +60,21 @@
 	"remote.extensionKind": {
 		"msjsdiag.debugger-for-chrome": "workspace"
 	},
-	"files.insertFinalNewline": true
+	"files.insertFinalNewline": true,
+	"cSpell.words": [
+		"SDKs",
+		"azcli",
+		"azurecli",
+		"datetime",
+		"dearmor",
+		"freebsd",
+		"gnupg",
+		"kube",
+		"kubeadm",
+		"kubectl",
+		"msiexec",
+		"openbsd",
+		"sunos",
+		"typemoq"
+	]
```

Comment:
Please don't include extra stuff like this in your PRs - we can look at including it separately but right now it's just clutter that isn't necessary and shouldn't be committed in the first place

Reply:
Why should this not be checked in? Doesn't it help everyone?

Reply:
I can separate it out into a separate PR but some words that are being added are in my PR.

Reply:
It only helps if we happen to use that particular extension - which I'm assuming most/all of us do not. I'd rather not have us cluttering up a common thing like this with something that most people don't need for what amounts to a pretty minor benefit

---
In reply to: [334567471](https://github.com/microsoft/azuredatastudio/pull/7647#discussion_r334567471) [](ancestors = 334567471)